### ISSUE TITLE
Easier to configure the limitation of ROS/DDS topic

### ISSUE THREAD

**Issue by evshary** [eclipse-zenoh/zenoh-plugin-dds#116]
### Describe the feature

When I try to integrate zenoh-bridge-dds into Autoware, I often want to constrain which topic can pass through zenoh-bridge-dds.
In my architecture, I want zenoh-bridge-dds acts like a gateway that transmits necessary ROS topic to the fleet management system, and the rest of the private ROS topic would not be seen by others.

The architecture will be something like the following:
![image](https://user-images.githubusercontent.com/456210/219528481-b5e7a97b-068d-4e97-a439-d6640e397727.png)

However, I found that the current ROS/DDS topic limitation is a little unfriendly.
In the description [here](https://github.com/eclipse-zenoh/zenoh-plugin-dds#limiting-the-ros2-topics-services-parameters-or-actions-to-be-routed), we need to use the regular expression to specify the allowed topic.
But while I have too many allowed topics and put all of them in one line, it's hard to maintain in [the configuration file](https://github.com/eclipse-zenoh/zenoh-plugin-dds/blob/master/DEFAULT_CONFIG.json5#L47).

IMHO, since we are using json in the configuration file, maybe it's suitable to use list instead of string to list all the allow / deny topic.

```json5
allow: [
  '/rosout',
  '/turtle1/cmd_vel',
  '/turtle1/rotate_absolute',
  ...
],
```

Maybe we can concatenate these topics with '|' internally, and no need to change the code too much.

**Comment by JEnoch** [comment]
Thanks for this suggestion! I think that's a good idea.
I'll try to do a PR for this in the coming weeks. Or maybe you would like to contribute ? 😉

**Comment by evshary** [comment]
> Thanks for this suggestion! I think that's a good idea. I'll try to do a PR for this in the coming weeks. Or maybe you would like to contribute ? wink

I'm fine with that. Maybe I can take a look at it these days. 😆

**Comment by JEnoch** [comment]
Implemented via #119.
Thanks for your contribution @evshary !
